app/styles/_ellipsis.less

```diff
@@ -133,3 +133,8 @@
     z-index: 1;
   }
 }
+
+// static loading... spacing
+.section-loading {
+  margin: 20px 0;
```

Comment:
> We usually try to position the loading message exactly where the content displays to avoid content jumping on the page as it loads. 

Except when we use the other "Loading..." message style.  Is there a rationale for having both?

![loading2](https://cloud.githubusercontent.com/assets/895728/20112641/2d5d93b6-a5bb-11e6-92b3-e7e3a6398345.png)